<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>2500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>2500</width>
    <height>1080</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gestion_inspection">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/gestion des inspections (1).png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="LOGO">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>55</y>
       <width>411</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/logo final.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_AJOUTER">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>830</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>AJOUTER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_SUPPRIMER">
     <property name="geometry">
      <rect>
       <x>1430</x>
       <y>850</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SUPPRIMR</string>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_MODIFIER">
     <property name="geometry">
      <rect>
       <x>1660</x>
       <y>850</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>MODIFIER</string>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_TRI">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>830</y>
       <width>81</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TRI</string>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_SMS">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>830</y>
       <width>91</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SMS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_REFRESH">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>220</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/images/REFRESH.png</normaloff>:/images/REFRESH.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QTableView" name="list">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>340</y>
       <width>1091</width>
       <height>451</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>1350</x>
       <y>830</y>
       <width>499</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/layout1.png</pixmap>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_id">
     <property name="geometry">
      <rect>
       <x>1480</x>
       <y>950</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_recherche">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>840</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_pdf">
     <property name="geometry">
      <rect>
       <x>1080</x>
       <y>220</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/images/pdf logo.jpg</normaloff>:/images/pdf logo.jpg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="mokabra">
     <property name="geometry">
      <rect>
       <x>1270</x>
       <y>840</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/Search.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>780</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>1160</x>
       <y>220</y>
       <width>48</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/images/excel.png</normaloff>:/images/excel.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="G_Inspection_NOTEPAD">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>830</y>
       <width>121</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>NOTEPAD</string>
     </property>
    </widget>
    <zorder>background</zorder>
    <zorder>LOGO</zorder>
    <zorder>G_Inspection_AJOUTER</zorder>
    <zorder>G_Inspection_TRI</zorder>
    <zorder>G_Inspection_SMS</zorder>
    <zorder>G_Inspection_REFRESH</zorder>
    <zorder>list</zorder>
    <zorder>label_8</zorder>
    <zorder>G_Inspection_SUPPRIMER</zorder>
    <zorder>comboBox_id</zorder>
    <zorder>lineEdit_recherche</zorder>
    <zorder>G_Inspection_MODIFIER</zorder>
    <zorder>pushButton_pdf</zorder>
    <zorder>mokabra</zorder>
    <zorder>label_32</zorder>
    <zorder>pushButton</zorder>
    <zorder>G_Inspection_NOTEPAD</zorder>
   </widget>
   <widget class="QGroupBox" name="Gestion_Inspection_ajouter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/ajouter inspection.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idin">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>360</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditfrais">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>520</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>709</x>
       <y>360</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>450</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>520</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frais du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>590</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Methode du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>670</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Etat du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>745</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Type du mission</string>
     </property>
    </widget>
    <widget class="QPushButton" name="return_ajouter">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>220</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ajouter_enregistrer">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>805</y>
       <width>205</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>434</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Methode">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>590</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Observation physique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Control sur pièces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sondage</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_Etat">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>670</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NON Acomplie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> Acomplie</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_Type">
     <property name="geometry">
      <rect>
       <x>858</x>
       <y>745</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Environnemental</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>De Qualite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Externe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Interne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Financier</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Modifier">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/modifier_inspection.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>440</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Date du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>735</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Type du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>660</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Etat du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>510</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Frais du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>580</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Methode du mission</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>729</x>
       <y>350</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditfrais_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>510</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Methode_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>580</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Observation physique</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Control sur pièces</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Sondage</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_idin_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>350</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>424</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Type_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>735</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Environnemental</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>De Qualite</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Externe</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Interne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Financier</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_Etat_m">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>660</y>
       <width>500</width>
       <height>48</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>NON Acomplie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string> Acomplie</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>810</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_annuler_m">
     <property name="geometry">
      <rect>
       <x>990</x>
       <y>810</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_sms">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/sms.png</pixmap>
     </property>
    </widget>
    <widget class="QLineEdit" name="msg">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>420</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>560</y>
       <width>431</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_SMS_confirme">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>810</y>
       <width>501</width>
       <height>161</height>
      </rect>
     </property>
     <property name="text">
      <string>CONFIRMER</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>410</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>560</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Telephone</string>
     </property>
    </widget>
    <widget class="QPushButton" name="return_sms">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/images/Vector.png</normaloff>:/images/Vector.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Notepad">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>2500</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1920</width>
       <height>1080</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="img.qrc">:/images/notePad.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_return_notepad">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>220</y>
       <width>80</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_enregistre_notepad">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>770</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="img.qrc">
       <normaloff>:/images/Button.png</normaloff>:/images/Button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>2000</width>
       <height>2000</height>
      </size>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_notepad">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>330</y>
       <width>421</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="NOtepad_afficher">
     <property name="geometry">
      <rect>
       <x>1100</x>
       <y>320</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>afficher</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>400</y>
       <width>1031</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Notepad_ecrire">
     <property name="geometry">
      <rect>
       <x>950</x>
       <y>330</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>ecrire</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>400</y>
       <width>1031</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_sms</zorder>
   <zorder>groupBox_Modifier</zorder>
   <zorder>Gestion_Inspection_ajouter</zorder>
   <zorder>gestion_inspection</zorder>
   <zorder>groupBox_Notepad</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
